conf/janus.transport.http.cfg.sample.in

```diff
@@ -15,7 +15,7 @@ port = 8088					; Web server HTTP port
 ;interface = eth0			; Whether we should bind this server to a specific interface only
 ;ip = 192.168.0.1			; Whether we should bind this server to a specific IP address (v4 or v6) only
 https = no					; Whether to enable HTTPS (default=no)
-;secure_port = 8889			; Web server HTTPS port, if enabled
+;secure_port = 8089			; Web server HTTPS port, if enabled
```

Comment:
well, I see the commit for this change says it syncs it with the demos, but the default in the transport plugin is 8889:
https://github.com/meetecho/janus-gateway/blob/master/transports/janus_http.c#L683